hifi_qt.py

```diff
@@ -74,10 +74,10 @@ def __init__(self, args):
                     self.qtUrl = self.assets_url + '/dependencies/vcpkg/qt5-install-5.12.6-ubuntu-19.10.tar.xz'
                 elif u_major > 18 and ( u_major != 19 and u_minor != 4):
                     print("We don't support " + distro.name(pretty=True) + " yet. Perhaps consider helping us out?")
-                    raise Exception('UNSUPPORTED LINUX VERSION!!!')
+                    raise Exception('YET UNSUPPORTED VERSION OF LINUX DISTRO!!!')
```

Comment:
```suggestion
                    raise Exception('UNSUPPORTED VERSION OF LINUX DISTRO!!!')
```
The "YET" doesn't make English sense.